package/boot/arm-trusted-firmware-mvebu/patches-mox-boot-builder/001-version.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/wtmi/Makefile b/wtmi/Makefile
+index 75754dc..3602ec3 100644
+--- a/wtmi/Makefile
++++ b/wtmi/Makefile
+@@ -41,7 +41,6 @@ else
+ 	LTO_FLAGS =
+ endif
+ 
+-override WTMI_VERSION = $(shell git describe --always --dirty --tags)
```

Comment:
Why do you remove this row? Do you have any plans to upstreaming this?

Reply:
Updated the commit messages, it does to mox-boot-builder what a04bffebbab51f7728e3bc2c5438f00508c5e708 does to a3700-utils and mv-ddr-marvell. Unsuitable for upstream since openwrt specific

Reply:
Let me try to understand this situation. You are doing some changes, which are not suitable for upstream, thus means your patch is a hack and it should not be merged here in the first place right? Because in other words, it means that we need to carry your patch forever. 

If I take a look at OpenWrt documentation for [patches](https://openwrt.org/docs/guide-developer/build-system/use-patches-with-buildsystem#naming_patches), I see these three  things:

```
- they must contain a proper subject
- they must contain a proper commit message explaining what they change
- they must contain a valid Signed-off-by line
```

This is not done. In the commit message, it would be helpful to what are you experiencing, like which error you met and why you solved it this way.

And my last thing which I want to highlight from the doc is this part:
> ALL patches must be in a way that they are potentially upstreamable,

Reply:
Right, but that doesn't match the practice as far as I can tell:
`find . -name \*.patch|xargs fgrep -Lw Subject`
(results are not 100% correct but you get the idea)

And that's in line with what I did here. Unsuitable for upstream doesn't automatically mean it's an unacceptable hack. Every distro has patches for most of its packages. It should aim to keep those as low as possible, but aiming for zero won't always be possible.

Reply:
It would help if you did not continue when others don't do it. Otherwise, we would not get from this state.